mysticeti-core/src/consensus/base_committer.rs

```diff
@@ -245,10 +252,26 @@ impl BaseCommitter {
     ) -> bool {
         let decision_blocks = self.block_store.get_blocks_by_round(decision_round);
 
+        // quickly reject if there isn't enough stake to support the leader from the potential certificates
+        let total_stake: Stake = decision_blocks
+            .iter()
+            .map(|b| self.committee.get_stake(b.author()).unwrap())
+            .sum();
```

Comment:
should we compute the total stake once and put it in as struct field?

Reply:
I think it's not a total stake here, but just stake of all decision_blocks

Reply:
Exactly